tsconfig.compodoc.json

```diff
@@ -20,16 +20,36 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "paths": {
-      "@spartacus/schematics": ["projects/schematics/index"],
-      "@spartacus/setup": ["core-libs/setup/public_api"],
-      "@spartacus/setup/ssr": ["core-libs/setup/ssr/public_api"],
-      "@spartacus/asm/assets": ["feature-libs/asm/assets/public_api"],
-      "@spartacus/asm/components": ["feature-libs/asm/components/public_api"],
-      "@spartacus/asm/core": ["feature-libs/asm/core/public_api"],
-      "@spartacus/asm": ["feature-libs/asm/public_api"],
-      "@spartacus/asm/occ": ["feature-libs/asm/occ/public_api"],
-      "@spartacus/asm/root": ["feature-libs/asm/root/public_api"],
-      "@spartacus/cart": ["feature-libs/cart/public_api"],
+      "@spartacus/schematics": [
+        "projects/schematics/index"
+      ],
+      "@spartacus/setup": [
+        "core-libs/setup/public_api"
+      ],
+      "@spartacus/setup/ssr": [
+        "core-libs/setup/ssr/public_api"
+      ],
+      "@spartacus/asm/assets": [
+        "feature-libs/asm/assets/public_api"
+      ],
+      "@spartacus/asm/components": [
+        "feature-libs/asm/components/public_api"
+      ],
+      "@spartacus/asm/core": [
+        "feature-libs/asm/core/public_api"
+      ],
+      "@spartacus/asm": [
+        "feature-libs/asm/public_api"
+      ],
+      "@spartacus/asm/occ": [
+        "feature-libs/asm/occ/public_api"
+      ],
+      "@spartacus/asm/root": [
+        "feature-libs/asm/root/public_api"
+      ],
+      "@spartacus/cart": [
+        "feature-libs/cart/public_api"
+      ],
```

Comment:
Small request, can you please format this file wiith vscode, I think the diff will be smaller. (and the merge simpler if there are conflicts)